SSPSolution/ResourceLib/ModelHandler.cpp

```diff
@@ -239,6 +233,14 @@ void Resources::ModelHandler::SetDevice(ID3D11Device * device)
  m_meshHandler->SetDevice(device);
  m_materialHandler->SetDevice(device);
  m_skeletonHandler->SetDevice(device);
+
+ ResourceContainer* temp;
+ unsigned int one = 2759249725;
+ unsigned int two = 3255160373;
+ FileLoader::GetInstance()->OpenFile(FileLoader::BPF_FILE);
```

Comment:
This is not a good idea. I suggest we open an issue on how to solve the loading of the character models